apps/frontend/src/modules/workspace/components/settings/Workspace/Billing.tsx

```diff
@@ -1,59 +1,109 @@
-import { planDetails } from "../../../utils";
+import { useAuth } from "@/modules/auth/hooks/useAuth";
+import { useWorkspace } from "@/modules/workspace/hooks/useWorkspace";
+import * as Progress from "@radix-ui/react-progress";
+import { BillingCategoriesEnum, planDetails } from "../../../utils";
+
+type BillingValues = {
+    count: number;
+    limit: number;
+};
+
+type ParsedBilling = {
+    events: BillingValues;
+    persons: BillingValues;
+    responses: BillingValues;
+};
+
 export const Billing = () => {
+    const { organizations } = useAuth();
+    const { workspace } = useWorkspace();
+
+    const currentWorkspace = organizations.find((org) => {
```

Comment:
Why do you need to do this? Is `workspace` from `useWorkspace` not enough to get `billingUsage`?